core/CMakeLists.txt

```diff
@@ -6,10 +6,23 @@ IF (NOT Kokkos_INSTALL_TESTING)
   ADD_SUBDIRECTORY(src)
 ENDIF()
 
+FUNCTION(KOKKOS_ADD_BENCHMARK_DIRECTORY DIR_NAME)
+  IF(NOT Kokkos_ENABLE_BENCHMARKS)
+    RETURN()
+  ENDIF()
+
+  IF(KOKKOS_HAS_TRILINOS)
+    message(
+      STATUS
```

Comment:
Should it be a warning?

Reply:
As long as skipping the benchmarks is not a big deal for Trilinos, `STATUS` should be fine.